### PR TITLE
AYS-444 & AYS-510 | UUID has been added to the response and time bug has been fixed

### PR DESCRIPTION
**Pull request by alpemreelmas**
### Checklist

Before submitting your pull request, ensure the following:

- [x] **Title and Branch Naming Conventions:**
    - The pull request title follows the
      standard: [Pull Request Naming Conventions](https://github.com/afet-yonetim-sistemi/ays-be/blob/main/CONTRIBUTING.md#pull-request-naming-conventions).
    - The branch name follows one of the conventions outlined in
      the [Branch Naming Conventions](https://github.com/afet-yonetim-sistemi/ays-be/blob/main/CONTRIBUTING.md#branch-naming-conventions).

- [x] **Local Testing:**
    - I have tested my changes locally on Postman, and they are working as expected.

- [x] **Code Quality:**
    - The code is formatted according to the project's coding guidelines and style.
    - The code has been reviewed to ensure its quality.
    - The code does not contain any issues flagged by SonarLint.

- [x] **Documentation:**
    - Necessary documentation has been added or existing documentation has been updated, specifically detailing changes made in Postman.

- [x] **Testing:**
    - Relevant unit tests have been written and included.
    - Relevant integration tests have been written and included.

- [x] **Reviewers and Assignees:**
    - Default reviewers have been assigned to this pull request.
    - Assignees have been added if necessary.

- [x] **Labels and Associations:**
    - No specific actions are required in the Labels and Associations section for this pull request.

### CODE REVIEW

**Review by MenekseYuncu — APPROVED** [review]

File: src/main/java/org/ays/common/model/response/AysResponse.java [hunk 1]
```diff
@@ -36,8 +43,11 @@ public class AysResponse<T> {
     /**
      * A pre-defined success response with no content.
      */
```
Comment (agitrubard) [inline src/main/java/org/ays/common/model/response/AysResponse.java:45]: Javadoc değişmemiş gibi görünüyor

**Review by agitrubard — CHANGES_REQUESTED** [review]
Javadoc'u da düzeltirsek bir sorun yok gibi görünüyor ✅

**Comment by agitrubard** [comment]
@alpemreelmas Javadoc'u düzenleyebilir misin? Sonrasında mergeleyelim.

File: src/main/java/org/ays/common/model/response/AysResponse.java (hunk 1 above)
Comment (alpemreelmas) [reply src/main/java/org/ays/common/model/response/AysResponse.java:45]:
```suggestion
 /**
     * A success response with no content.
     */
```

Uygun mudur bu şekilde ?

Comment (agitrubard) [reply src/main/java/org/ays/common/model/response/AysResponse.java:45]: Uygundur eline sağlık ✅

**Review by agitrubard — DISMISSED** [review]

**Pull request merged**